packages/ui/src/common/components/page/Sidebar/Breadcrumbs/BreadcrumbsList.tsx

```diff
@@ -1,17 +1,27 @@
 import React from 'react'
+import { useHistory } from 'react-router'
 import styled from 'styled-components'
+import useBreadcrumbs from 'use-react-router-breadcrumbs'
 
-import { BreadcrumbsItem, BreadcrumbsItemProps } from './BreadcrumbsItem'
+import { knownRoutes, excludePaths } from '../../../../model/breadcrumbs'
+
+import { BreadcrumbsItem } from './BreadcrumbsItem'
 
 export interface BreadcrumbsListProps {
-  crumbs: Array<BreadcrumbsItemProps>
+  objectName?: string
 }
 
-export function BreadcrumbsList({ crumbs }: BreadcrumbsListProps) {
+export function BreadcrumbsList({ objectName }: BreadcrumbsListProps) {
+  const { location } = useHistory()
+  const routesList = objectName ? [...knownRoutes, { path: location.pathname, breadcrumb: objectName }] : knownRoutes
+  const crumbs = useBreadcrumbs(routesList, { excludePaths })
+  console.log(crumbs)
```

Comment:
```suggestion
  const crumbs = useBreadcrumbs(routesList, { excludePaths })
```

:oncoming\_police\_car: